code/modules/atmospherics/machinery/air_alarm/_air_alarm.dm

```diff
@@ -568,7 +569,8 @@ GLOBAL_LIST_EMPTY_TYPED(air_alarms, /obj/machinery/airalarm)
 	danger_level = max(danger_level, tlv_collection["pressure"].check_value(pressure))
 	danger_level = max(danger_level, tlv_collection["temperature"].check_value(temp))
 	if(total_moles)
-		for(var/gas_path in GLOB.meta_gas_info)
+		var/list/cached_gas_info = GLOB.meta_gas_info
+		for(var/datum/gas/gas_path as anything in cached_gas_info)
```

Comment:
this doesnt do anything, the right-hand-side of `in` in this context is evaluated once, and it will copy the list then iterate over it.